agent-core/src/test/java/com/newrelic/agent/android/test/stub/StubAgentImpl.java

```diff
@@ -179,11 +180,21 @@ public long getSessionDurationMillis() {
 
     @Override
     public boolean hasReachableNetworkConnection(String reachableHost) {
-        return true;
+        return reachableHost == null;
```

Comment:
For the stub (used mostly in testing) we assume reachability. Why the change here?

Reply:
It was changed due to testing purpose, if I send in something is not null, I can simulate it has network connection or not.